src/lurch.c

```diff
@@ -1806,7 +1806,11 @@ static void lurch_message_decrypt(PurpleConnection * gc_p, xmlnode ** msg_stanza
     goto cleanup;
   }
   if (!key_p) {
+    const char *to = room_name ?: sender;
```

Comment:
I guess I'll have to review the whole error printing in this function - the cleanup code always presents the error to the conversation with the `sender`, which for MUCs does not make sense.